web/src/main/java/hu/psprog/leaflet/lms/web/controller/DocumentsController.java

```diff
@@ -167,12 +164,15 @@ public ModelAndView processDeleteDocument(@PathVariable(PATH_VARIABLE_ID) Long d
      * Processes document status change request.
      *
      * @param documentID ID of the document the status to be changed of
+     * @param redirectTo value where redirect after the change to
      * @param redirectAttributes redirection attributes
      * @return populated {@link ModelAndView} object (redirection to view the edited document)
      * @throws CommunicationFailureException on Bridge communication failure
      */
     @RequestMapping(method = RequestMethod.POST, path = PATH_STATUS)
-    public ModelAndView processChangeDocumentStatus(@PathVariable(PATH_VARIABLE_ID) Long documentID, RedirectAttributes redirectAttributes)
+    public ModelAndView processChangeDocumentStatus(@PathVariable(PATH_VARIABLE_ID) Long documentID,
+                                                    @RequestParam(REQUEST_PARAM_REDIRECT) String redirectTo,
```

Comment:
Why is this needed?

Reply:
Because I use this on both document list and view pages. After the button click it should redirect to the correct page. I used a hidden input value to determine which page is needed.

Reply:
Ah okay, thanks!